### PR TITLE
get reference to fastlane folder the more correct way

### PR DESCRIPTION
**Pull request by taquitos**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Description
Update to get the fastlane folder the more correct way (that will even find it if the folder starts with a .)

### CODE REVIEW

File: fastlane_core/lib/fastlane_core/helper.rb [hunk 1]
```diff
@@ -240,6 +240,10 @@ def self.fastlane_enabled?
       @enabled ||= !FastlaneCore::FastlaneFolder.path.nil?
     end
 
+    def self.fastlane_enabled_folder_path
```
Comment (KrauseFx) [inline fastlane_core/lib/fastlane_core/helper.rb]: A comment here would be great to describe what the method does

**Review by KrauseFx — APPROVED** [review]
Wohooo, all the clean ups :recy

**Review by ohayon — APPROVED** [review]
💯

**Review by KrauseFx — APPROVED** [review]
:fidget_spinner:

**Comment by taquitos** [comment]
@KrauseFx omg when are we getting fastlane fidget spinners?

**Pull request merged**